noir-projects/noir-contracts/contracts/fpc_contract/src/main.nr

```diff
@@ -110,7 +106,7 @@ pub contract FPC {
     // docs:start:complete_refund
     #[public]
     #[internal]
-    fn complete_refund(accepted_asset: AztecAddress, refund_slot: Field, max_fee: u128) {
+    fn _complete_refund(accepted_asset: AztecAddress, refund_slot: Field, max_fee: u128) {
```

Comment:
Sneaked this naming change in. I think it's cute to have internal funcs with underscore.